src/configs.rs

```diff
@@ -16,14 +16,42 @@ pub(crate) struct Opts {
     #[clap(long)]
     pub debug: bool,
     /// AWS S3 bucket name to get the stream from
-    #[clap(long)]
+    #[clap(long, env = "s3_bucket_name")]
     pub s3_bucket_name: String,
+    /// AWS Access Key with the rights to read from AWS S3
+    #[clap(long, env = "NEAR_LAKE_AWS_KEY")]
```

Comment:
we have to pass in aws credentials through .env file so we can inject it form cloudrun